install_config/cluster_metrics.adoc

```diff
@@ -112,11 +112,11 @@ API
 ----
 
 . Before it can deploy components, the *metrics-deployer* service account must
-also be granted the `edit` permission for the *openshift-infra* project:
+also be granted the `edit` permission for the *openshift-metrics* project:
```

Comment:
transitively, because openshift-infra contains many service accounts with elevated permissions, and the edit role in that namespace allows reading all of their secrets